### ISSUE TITLE
[Reality Module] Encourage safe parameter configuration

### ISSUE THREAD

**Issue by manboy-eth** [gnosis/zodiac-safe-app#114]
We should encourage users to set safe values for Module timouts (days not hours) and bond (an attacker should have something at risk).

More info: https://twitter.com/auryn_macmillan/status/1576674887384829952

**Comment by juliopavila** [comment]
It's essential to add this solution that I took from [here](https://twitter.com/auryn_macmillan/status/1576675621228670976?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1576675621228670976%7Ctwgr%5E5ca4c91c7d11ef4229e12e67fa431c9c8565793f%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fpublish.twitter.com%2F%3Fquery%3Dhttps3A2F2Ftwitter.com2Fauryn_macmillan2Fstatus2F1576675621228670976widget%3DTweet):

```
To recap, in order to securely use a Reality module, a DAO must:
1. have multiple independent parties monitoring calls to addProposal().
2. have a long questionTimeout for people to post bonds (48+ hours).
3. have a high enough minimumBond to disincentivize malicious proposals.
```
